src/ripple/overlay/impl/PeerImp.cpp

```diff
@@ -1095,7 +1097,7 @@ PeerImp::onMessage (std::shared_ptr <protocol::TMTransaction> const& m)
                 flags |= SF_TRUSTED;
             }
 
-            if (! app_.config().VALIDATION_PRIV.isSet())
+            if (! app_.config().VALIDATION_PUB.size())
```

Comment:
`size` is used in several places to check if a key is valid or not. Consider adding an `empty` function.